documentation/projects.md

```diff
@@ -140,20 +144,23 @@ The following are common files and directories found in a Bolt project.
 |[`inventory.yaml`](inventory_file_v2.md)|Contains a list of known targets and target specific data.|
 |[`plans/`](plans.md)|A directory for storing your plans.|
 |[`tasks/`](tasks.md)|A directory for storing your tasks.|
-|`files/`| A directory for storing content consumed by your tasks and plans, such as scripts.|
-|[`Puppetfile`](bolt_installing_modules.md#)|Specifies which modules to install for the project. 🧪 **Experimental:** If you've opted in to the experimental module management workflows, Bolt manages this file. Avoid editing it. For more information, see [Managing modules](managing_modules.md).|
-|[`modules/`](bolt_installing_modules.md#)|The directory where modules from the `Puppetfile` are installed. In most cases, do not edit these modules locally.|
-|[`site-modules/`](bolt_installing_modules.md)|Local modules that are edited and versioned with the Bolt directory.|
+|`files/`|A directory for storing content consumed by your tasks and plans, such as scripts.|
```

Comment:
Can't make a suggestion since it's too far up, but the 'Directory' column should be renamed to 'Directory/File' as well.